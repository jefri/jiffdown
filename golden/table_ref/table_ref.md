| Name | Value    |
| ---- | -------- |
| Cell | &{#ref}; |

# ref

A section
