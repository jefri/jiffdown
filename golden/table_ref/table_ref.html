<table>
<thead>
<tr>
<th>Name</th>
<th>Value</th>
</tr>
</thead>
<tbody><tr>
<td>Cell</td>
<td><h1 id="ref">ref</h1>
<p>A section</p>
</td>
</tr>
</tbody></table>
